lib/source-map-url.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+// Forked from https://github.com/lydell/source-map-url/blob/master/source-map-url.js
+// We use a slightly adjusted regex here
+const convertSourceMap = require('convert-source-map');
```

Comment:
any way we can use convert-source-map directly instead of needing teh lib/source-map-url.js file?

Reply:
It seems to work a bit differently, and I didn't find a good method it exposed (?) to just get all source map URLs in a file - there is a method to just get the last one, and a method to remove _all_ of them in the file, both of which are not suited to what we need here 😓